test/metabase/driver/sql/parameters/substitute_test.clj

```diff
@@ -167,16 +167,15 @@
                    (substitute query {"param" (i/map->FieldFilter
                                                {:field (Field (mt/id :venues field))
                                                 :value {:type  operator
-                                                        :value value}})}))))))))
-
+                                                        :value value}})})))))))))
 
 ;;; -------------------------------------------- Referenced Card Queries ---------------------------------------------
 
 (deftest substitute-referenced-card-query-test
   (testing "Referenced card query substitution"
     (let [query ["SELECT * FROM " (param "#123")]]
       (is (= ["SELECT * FROM (SELECT 1 `x`)" []]
-             (substitute query {"#123" (i/map->ReferencedCardQuery {:card-id 123, :query "SELECT 1 `x`"})})))))))
```

Comment:
Unrelated to this fix but this `deftest` was actually inside of the parent test form. Fixed the parens.